server/dfr3-service/src/main/java/edu/illinois/ncsa/incore/service/dfr3/models/DFR3Set.java

```diff
@@ -34,6 +34,8 @@
 
     protected String creator;
 
```

Comment:
It's not related to this task, at some point we maybe able to get rid of the resultUnit and returnType fields since it's now moved the "returnType" inside each curve